x/evm/keeper/msg_server.go

```diff
@@ -122,7 +122,7 @@ func (s msgServer) ConfirmGatewayDeployment(c context.Context, req *types.Confir
 			sdk.NewAttribute(types.AttributeKeyAddress, req.Address.Hex()),
 			sdk.NewAttribute(types.AttributeKeyBytecodeHash, hex.EncodeToString(crypto.Keccak256(deploymentBytecode))),
 			sdk.NewAttribute(types.AttributeKeyConfHeight, strconv.FormatUint(height, 10)),
-			sdk.NewAttribute(types.AttributeKeyPoll, string(types.ModuleCdc.MustMarshalJSON(&pollKey))),
+			sdk.NewAttribute(types.AttributeKeyPoll, pollKey.String()),
```

Comment:
Please revert. Events do not serve humans as their top priority but code, and this is going to break vald.